src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int amountBuy = 0;
+        int amountSupply = 0;
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                String[] parts = line.split(",");
+                String operationType = parts[0];
+                int amount = Integer.parseInt(parts[1]);
+
+                if (operationType.equals("buy")) {
+                    amountBuy += amount;
+                } else if (operationType.equals("supply")) {
+                    amountSupply += amount;
+                }
+            }
+        } catch (IOException e) {
```

Comment:
Consider using a constant for the delimiter "," used in `split`, to make the code more readable and maintainable.